retail/interactive-tutorial/RetailSearch.Samples/SearchWithFacetSpec.cs

```diff
@@ -0,0 +1,116 @@
+﻿// Copyright 2021 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// [START retail_search_product_with_facet_spec]
+
+using Google.Cloud.Retail.V2;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Serialization;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+/// <summary>
+/// Search with facet spec sample class.
+/// </summary>
+public static class SearchWithFacetSpec
```

Comment:
Is the class being static and everything else for that matter a requisite of the tutorials?

Reply:
Divided into two separate classes by suggestion below (Sample class and Tutorial class - the static one)